workspace-manager-clienttests/src/main/java/scripts/testscripts/ServiceStatus.java

```diff
@@ -0,0 +1,30 @@
+package scripts.testscripts;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.api.UnauthenticatedApi;
+import bio.terra.workspace.client.ApiClient;
+import bio.terra.workspace.model.SystemStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.utils.WorkspaceManagerServiceUtils;
+
+public class ServiceStatus extends TestScript {
+    private static final Logger logger = LoggerFactory.getLogger(ServiceStatus.class);
+
+    /** Public constructor so that this class can be instantiated via reflection. */
+    public ServiceStatus() {
+        super();
+    }
+
+    @Override
+    public void userJourney(TestUserSpecification testUser) throws Exception {
+        ApiClient apiClient = WorkspaceManagerServiceUtils.getClient(server);
+        UnauthenticatedApi unauthenticatedApi = new UnauthenticatedApi(apiClient);
+        SystemStatus systemStatus = unauthenticatedApi.serviceStatus();
+        logger.info("systemStatus: {}", systemStatus);
+
+        int httpCode = unauthenticatedApi.getApiClient().getStatusCode();
+        logger.info("Service status HTTP code: {}", httpCode);
```

Comment:
in a concurrent perf test, this is going to log a lot of data, and the two log lines can get interleaved with other parallel requests. Combine into one?
```
logger.info("Service status with code {}: {}", httpCode, systemStatus);
```

Reply:
Merged multiple logs into one statement.